docs/clink.md

```diff
@@ -1279,17 +1279,13 @@ The [clink-flex-prompt](https://github.com/chrisant996/clink-flex-prompt) script
 
 ### oh-my-posh
 
-The [oh-my-posh](https://github.com/JanDeDobbeleer/oh-my-posh3) program can generate fancy prompts.  Refer to its documentation for how to configure it.
+The [oh-my-posh](https://github.com/JanDeDobbeleer/oh-my-posh) program can generate fancy prompts. Refer to its [documentation](https://ohmyposh.dev) for how to configure it.
 
-Integrating oh-my-posh with Clink is easy:  just save the following text to an `oh-my-posh.lua` file in your Clink scripts directory (run `clink info` to find that), and make sure the oh-my-posh.exe program is in a directory listed in the %PATH% environment variable.  (Or edit the script below to provide a fully qualified path to the oh-my-posh.exe program.)
+Integrating oh-my-posh with Clink is easy: just save the following text to an `oh-my-posh.lua` file in your Clink scripts directory (run `clink info` to find that), and make sure the `oh-my-posh.exe` program is in a directory listed in the `%PATH%` environment variable (or edit the script below to provide a fully qualified path to the oh-my-posh.exe program). Replace the config while with your own configuration and you're good to go.
 
 ```lua
 -- oh-my-posh.lua
-local ohmyposh_prompt = clink.promptfilter(1)
-function ohmyposh_prompt:filter(prompt)
-    prompt = io.popen("oh-my-posh.exe"):read("*a")
-    return prompt, false
-end
+load(io.popen("oh-my-posh.exe --config=\"C:/Users/me/jandedobbeleer.omp.json\" --init --shell cmd"):read("*a"))()
```

Comment:
Is that a stray/redundant `()` at the end of the line?

nit:  For slightly simpler string maintenance, the inner quotes don't need to be escaped if the outer quotes are single quotes.  E.g.
```lua
load(io.popen('oh-my-posh.exe --config="C:/Users/me/jandedobbeleer.omp.json" --init --shell cmd'):read("*a"))
```

Reply:
The `()` at the end, as far I understood, was to run the code right after `load` (I understood it's a function). I can check if that's necessary or not.

Reply:
@chrisant996 confirmed that the `()` is needed as it only runs the code that way.